sormas-app/app/src/main/java/de/symeda/sormas/app/task/edit/TaskEditFragment.java

```diff
@@ -165,6 +173,19 @@ protected void onAfterLayoutBinding(FragmentTaskEditLayoutBinding contentBinding
 				contentBinding.taskButtonPanel.setVisibility(GONE);
 			}
 		}
+
+		contentBinding.taskAssigneeUser.addValueChangedListener(v -> {
+			User assigneeUser = (User) v.getValue();
+			if (initialAssigneeUser == null) {
```

Comment:
you could merge this into initialAssigneeUser == null || !initialAssigneeUser.equals(assigneeUser)
would be shorter and cleaner
setting the initial value to the same value seems redundant